analysis/pcmdimetrics/portrait4seasv1p1p1.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+#################################################################################
+#  RELIES UPON CDAT MODULES 
+#  PRODUCE SIMPLE PORTRAIT PLOT BASED ON RESULTS FROM PCMDI'S METRICS PACKAGE 
+#  WRITTEN BY P. GLECKLER, PCMDI/LLNL
+#  FURTHER TWEAKED BY C. DOUTRIAUX
+#  ISSUE TO BE RESOLVED WITH UV-CDAT PLANNED IMPROVEMENTS: ABILITY TO SAVE PLOTS IN MULTIPLE FORMATS  
+#################################################################################
+
+# CHECK TO VERIFY VCS MODULE IS INSTALLED
+try:
+    import vcs
+except:
+    raise RuntimeError("Sorry your python is not build with VCS support and cannot generate portrait plots")
+
+# STANDARD PYTHON MODULES
+import glob
+import json
+import os
+import sys
+import re
+import numpy as np
```

Comment:
A similar check should be done for other non-standard python modules (like numpy, MV2, pcmdi_metrics).

Reply:
@underwoo new commits are in. If there are more changes and improvements needed, perhaps I could address it later after the initial push. Thank you.